rust.makefile

```diff
@@ -62,7 +62,7 @@ lint: version
 	# to be sure clippy is actually run
 	touch src/lib.rs
 	mkdir -p target
-	cargo clippy --message-format json --all-targets --examples --tests > target/cargo-clippy.json
+	cargo clippy --message-format json --all-targets --examples --tests -- --deny warnings > target/cargo-clippy.json
```

Comment:
the first -- with nothing is wanted ?

Reply:
yes, to mean the end of clippy options (and the remaining stuff is passed to the compiler)